backend/routes/room_routes.py

```
@@ -122,7 +121,7 @@ def add_room():
                 'error_details': str(e),
                 'operation_result': 'failed'
             })
-            return jsonify({"error": f"Room creation failed due to constraint violation: {str(e)}"}), 400
+            return error_response(f"Room creation failed due to constraint violation: {str(e)}", 400)
```

Ruff (0.15.0)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)